apps/web/src/pages/TalentEvents/TalentEvent/util.tsx

```diff
@@ -0,0 +1,121 @@
+import { IntlShape } from "react-intl";
+import { JSX } from "react";
+
+import { notEmpty, uniqueItems } from "@gc-digital-talent/helpers";
+import {
+  TalentEventNominationsTableFragment as TalentEventNominationsTableFragmentType,
+  TalentNominationGroupStatus,
+} from "@gc-digital-talent/graphql";
+import { Chip, Link } from "@gc-digital-talent/ui";
+
+import { getFullNameLabel } from "~/utils/nameUtils";
+import useRoutes from "~/hooks/useRoutes";
+import messages from "~/messages/talentNominationMessages";
+
+import { TalentNominator } from "./types";
+
+export function nominationsToNominators(
+  nominations: TalentEventNominationsTableFragmentType["nominations"],
+): TalentNominator[] {
+  const nominationsFiltered = nominations?.filter(notEmpty) ?? [];
+  const nominators = nominationsFiltered.map(
+    (nomination) => nomination?.nominator,
+  );
+  const nominatorsFiltered = nominators?.filter(notEmpty) ?? [];
+
+  return nominatorsFiltered;
+}
+
+export function nominatorsAccessor(
+  nominators: TalentNominator[],
+  intl: IntlShape,
+): string {
+  const nominatorsSorted = nominators.sort((a, b) => {
+    return (
+      (a.lastName ?? "").localeCompare(b.lastName ?? "") ||
+      (a.firstName ?? "").localeCompare(b.firstName ?? "")
+    );
+  });
+
+  const nominatorsNamesArray = nominatorsSorted.map((nominator) =>
+    getFullNameLabel(nominator?.firstName, nominator?.lastName, intl),
+  );
+  const uniqueNamesToDisplay = uniqueItems(nominatorsNamesArray);
+
+  return uniqueNamesToDisplay.join(", ");
+}
+
+export function statusCell(
+  status: TalentEventNominationsTableFragmentType["status"],
+): JSX.Element | null {
+  if (!status?.value || !status?.label?.localized) {
+    return null;
+  }
+
+  if (status.value === TalentNominationGroupStatus.Approved) {
+    return <Chip color="success">{status.label.localized}</Chip>;
+  }
+
+  if (status.value === TalentNominationGroupStatus.PartiallyApproved) {
+    return <Chip color="success">{status.label.localized}</Chip>;
+  }
+
+  if (status.value === TalentNominationGroupStatus.InProgress) {
+    return <Chip color="primary">{status.label.localized}</Chip>;
+  }
+
+  if (status.value === TalentNominationGroupStatus.Rejected) {
+    return <Chip color="error">{status.label.localized}</Chip>;
+  }
+
+  return null;
+}
+
+export const nomineeNameCell = (
+  eventId: string,
+  nominationGroupId: string,
+  nominee: TalentEventNominationsTableFragmentType["nominee"],
+  paths: ReturnType<typeof useRoutes>,
+  intl: IntlShape,
+) => {
+  const nomineeName = getFullNameLabel(
+    nominee?.firstName,
+    nominee?.lastName,
+    intl,
+  );
+
+  return (
+    <Link href={paths.talentNominationGroupProfile(eventId, nominationGroupId)}>
+      {nomineeName}
+    </Link>
+  );
```

Comment:
A little wonky at the moment due to https://github.com/GCTC-NTGC/gc-digital-talent/issues/13148

Reply:
Okay, resolved now. Link fixed again in https://github.com/GCTC-NTGC/gc-digital-talent/pull/13151/commits/62e1dc315cbf6ec952db0e59c7acef61ddb846d7
Points to where intended https://github.com/GCTC-NTGC/gc-digital-talent/issues/13122#issuecomment-2773080860